src/SupervisionSuccess/model/produceProjections.js

```diff
@@ -171,7 +175,7 @@ function produceProjections(
 
   return {
     chartData,
-    savings: totalSavings > 0.0001 || totalSavings < 0 ? Number(totalSavings.toFixed(6)) : 0,
+    savings: Number(totalSavings.toFixed(3)),
```

Comment:
@jessex @cawarren The problem with 0 behavior comes from total savings and its accuracy, but error does not exceed 0.0005 (for now, at least). At the same time we do not use numbers smaller than 0.001 for savings, so using `toFixed(3)` instead `toFixed(6)` has the meaning. We can do so, or use the condition commented above.

Reply:
I think this approach you have now seems good for the use case. Thanks, @nasaownsky!